src/solver/variable/state.cpp

```diff
@@ -73,14 +76,14 @@ State::State(Data::Study& s) :
 {
 }
 
-void State::initFromThermalClusterIndex(const uint clusterAreaWideIndex, uint numSpace)
+void State::initFromThermalClusterIndex(const uint clusterAreaWideIndex)
```

Comment:
variable **thermalClustersProductions** is initialized in a **if... else...** statement, but it's not clean : 
```C++
if (/*condition 1*/)
{
   ... 
   // giving a value to thermalClustersProductions
}
else
{
    // giving another value to thermalClustersProductions
}
```
We should initialize it to zero : `thermalClustersProductions.resize(count, 0.);`